ru-ru/[pyqt-ru] New translation

```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
```

Comment:
Just `pyqt-ru.html.markdown`, no `[pyqt-ru]` prefix, as you can see from the other files in that directory.